src/main/scala/dhall/utilities/TypeLevelFunctions.scala

```diff
@@ -0,0 +1,10 @@
+package dhall.utilities
+
+import language.higherKinds
```

Comment:
If we use higherKinds elsewhere as well then we should enable it from sbt I think.